README.md

```diff
@@ -99,6 +99,7 @@ Most or all of these should be fixed in the generator over time.
 - `Array` changed to `Iterable` for WebIDL `sequence`s in argument positions (but not in return positions).
 - `any` changed to `object` for WebIDL `object`.
 - `| SharedArrayBuffer` added for `[AllowShared] BufferSource`.
+- `| null` changed to `| null | undefined` for WebIDL `?` items in `sequence`s.
```

Comment:
I [double checked](https://webidl.spec.whatwg.org/#js-nullable-type) and actually all WebIDL nullable `?` items should accept `| null | undefined`.
There are two others that are not in sequences:
- setBindGroup (both overloads)
- setVertexBuffer

Chrome's behavior confirms this: it allows
`setBindGroup(0, null)`
`setBindGroup(0, undefined)`
but not
`setBindGroup(0)`

Reply:
I'll take care of this.